Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'haml'
 gem 'haml-rails'
 gem 'icalendar'
 gem 'jquery-rails'
-gem 'jquery-ui-rails'
+gem 'jquery-ui-rails', '~>5.0.5'
```

Comment:
Unless someone told you to do this, this seems like something to add in a separate PR.